src/UI/Factory.php

```diff
@@ -40,11 +44,16 @@
 	 *     1: Icons MUST be used to represent objects or context.
 	 *     2: Icons MUST be used in combination with a title or label.
 	 *     3: An unique Icon MUST always refer to the same thing.
+	 *     4: Preconditioned objects whose precondition is not met yet
+	 *        feature Disabled Icons
+	 *     5: Objects to which a user has no read access features Disabled Icons.
 	 *   style:
 	 *     1: Icons MUST have a class indicating their usage.
 	 *     2: Icons MUST be tagged with a CSS-class indicating their size.
+	 *     3: Disabled Icons will be shaded grey.
```

Comment:
This is no rule (not MUST, SHOULD, ...), also composition already states something similar.